test/system/webhooks_test.rb

```diff
@@ -0,0 +1,56 @@
+require 'application_system_test_case'
+
+class WebhooksTest < ApplicationSystemTestCase
+  # playback sucessfully created
```

Comment:
We can probably get rid of these comments, and the file could be DRY'd up by using a `setup` block to create and instance variable e.g. `@body = {...}` to be reused between tests. You might also want get rid of `send_webhook_request` and call `webhook_request` directly in setup, leave that decision to you.